kernel/packages/shared/comms/index.ts

```diff
@@ -785,7 +882,7 @@ export async function connect(userId: string) {
           },
           authHandler: async (msg: string) => {
             try {
-              return Authenticator.signPayload(getIdentity(), msg)
+              return Authenticator.signPayload(getIdentity() as AuthIdentity, msg)
```

Comment:
Why is that explicit `as` needed?

Reply:
oh. null?

Reply:
So it seems. I don't actually remember. I think I may have been more focused on making it compile than on why it didn't.